### ISSUE TITLE
MediaStore not updated

### ISSUE THREAD

**Issue by Devon22** [syncthing/syncthing-android#1818]
The picture is deleted on the PC, and after syncing to Android, the MediaStore is not updated, and the result is shown in the following figure.

![Screenshot_20220806-171206_Gallery](https://user-images.githubusercontent.com/95129064/183242956-3ee77e03-a1f5-427d-b487-cc7571de7d04.jpg)

Version Information

    App Version: 1.20.4
    Syncthing Version: v1.20.4
    Android Version: Android 12

**Comment by tomasz1986** [comment]
Probably related to / caused by https://github.com/syncthing/syncthing-android/pull/1803.

**Comment by imsodin** [comment]
Yes it is. As I wrote there,  I am very interested in hearing about methods to let androids mediastore know about a deletion or let it check for a path, or directory content - really anything that makes it pick up the deletion without actually scheduling a deletion.  
One thing to try is using the `.scanFile` method, but all (inofficial) info I found indicates that's not going to do anything - so my motivation to test is quite low. So hints/contributions welcome.

**Comment by steelbytes** [comment]
> Yes it is. As I wrote there, I am very interested in hearing about methods to let androids mediastore know about a deletion or let it check for a path, or directory content - really anything that makes it pick up the deletion without actually scheduling a deletion. One thing to try is using the `.scanFile` method, but all (inofficial) info I found indicates that's not going to do anything - so my motivation to test is quite low. So hints/contributions welcome.

stackoverflow says that you call MediaScannerConnection.scanFile with the name of the file that was just deleted.  yes it seems counter intuitive at first but give it a go

**Comment by Devon22** [comment]
Is it possible to fix this problem?
Version 1.23.4 still has the same problem.
